chromatic/rainbows/actions/conversions.py

```diff
@@ -0,0 +1,79 @@
+from ...imports import *
+
+__all__ = ["to_nparray", "to_df"]
+
+
+def to_nparray(self, timeformat='h'):
+    """ Convert Rainbow object to 1D and 2D numpy arrays
+    Parameters
+    ----------
+        self : Rainbow object
+            chromatic Rainbow object to convert into array format
+        timeformat : str
+            (optional, default='hours')
+            The time format to use (seconds, minutes, hours, days etc.)
+    Returns
+    ----------
+        rflux : np.array
+            flux (MJy/sr)        [n_wavelengths x n_integrations]
+        rfluxe : np.array
+            flux error (MJy/sr)  [n_wavelengths x n_integrations]
```

Comment:
This is a very tiny fussy thing, but could we use "uncertainty" instead of "error"? It's totally just word-smithing, but to me "uncertainty" feels more accurate ("we have a measurement with some known uncertainty associated with it") than "error" (which reads to me more as "we have a measurement but there's something wrong with it"). To me, an "error" can be fixed, while an "uncertainty is just a fundamental part of science? (Again, super not crucial!)

Reply:
Done!